docs/proposals/network/README.md

```diff
@@ -0,0 +1,66 @@
+# Software Defined Network
+
+## Analysis
+
+### Examples
+
+- Canal
+- Calico
+- Flannel
+- Weave
+- Cilium
+
+![benchmark-cni](https://miro.medium.com/v2/resize:fit:1400/1*zDiFQZVXuL923x0LbmXMaA.png)
+
+### Sources
+
+- [CNI](https://github.com/containernetworking/cni) also makes a good framework for creating a new container networking project from scratch
+- [Networks - treeptik k8s](https://treeptik.gitbook.io/k8s/fundamentals/)network
+- [Explain NS linux](https://www.youtube.com/watch?v=j_UUnlVC2Ss)
+- [Spec - CNI](https://github.com/containernetworking/cni/blob/main/SPEC.md)
+
+<br>
+
+## Network domain
+
+### Definition
+
+The goal is to be able to:
+
+- manage networks through the software
+- isolate instance/pods with isolated networks
+- apply network security policies
+
+Options:
```

Comment:
FWIW, this should not be part of your final version. You have to make a decision about that as part of this architecture document.

Reply:
We have specified with a `Scope` and `Out of scope`, tell us if it's better